pkg/controllers/vdb/offlineupgrade_reconciler.go

```diff
@@ -163,7 +163,9 @@ func (o *OfflineUpgradeReconciler) finishUpgrade(ctx context.Context) (ctrl.Resu
 // a method other than offline or auto.  We can fall into this codepath if we
 // are running a version of Vertica that doesn't support online upgrade.
 func (o *OfflineUpgradeReconciler) logEventIfThisUpgradeWasNotChosen(_ context.Context) (ctrl.Result, error) {
-	o.Manager.logEventIfRequestedUpgradeIsDifferent(vapi.OfflineUpgrade)
+	if o.PFacts.GetSandboxName() == vapi.MainCluster {
```

Comment:
Why do we need this change? Don't we support online upgrade in a sandbox?

Reply:
Because this logs a message when the selected upgrade policy is different from the one specified in VDB. But we only support offline upgrade for sandbox so we don't need to log this message.